ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml

```diff
@@ -76,6 +76,10 @@ tests:
   openshift_installer_custom_test_image:
     cluster_profile: aws
     from: stable:cluster-logging-operator-e2e
+- as: e2e-upgrade
+  commands: hack/testing-olm-upgrade/test-030-olm-upgrade-n-1-n.sh
+  openshift_installer_src:
+    cluster_profile: aws
```

Comment:
Please don't add new jobs using the template and instead use the step-based workflow:
```
- as: your test
  steps:
    cluster_profile: aws
    test:
    - as: test
      cli: latest
      commands: command
      from: src
      resources:
        requests:
          cpu: 100m
    workflow: ipi-aws
```

Reply:
there isn't a workflow yet available for this though, in fact, we still need to create/migrate all of our repo's tests to the step registry model